web/ui/react-app/src/pages/flags/Flags.tsx

```diff
@@ -106,10 +106,10 @@ export const FlagsContent: FC<FlagsProps> = ({ data = {} }) => {
             const sanitizeOpts = { allowedTags: ['strong'] };
             return (
               <tr key={flagMatchStr}>
-                <td>
+                <td id="flag-item">
```

Comment:
Now you are reusing the same `id` multiple times on the same page... it should probably still be a class, as before?